airsonic-main/src/main/webapp/WEB-INF/jsp/albumMain.jsp

```diff
@@ -245,21 +245,21 @@
 
     <c:if test="${not empty model.artist and not empty model.album}">
         <sub:url value="https://www.google.com/search" var="googleUrl" encoding="UTF-8">
-            <sub:param name="q" value="\"${model.artist}\" \"${model.album}\""/>
+            <sub:param name="q" value="\"${fn:escapeXml(model.artist)}\" \"${fn:escapeXml(model.album)}\""/>
         </sub:url>
         <sub:url value="https://en.wikipedia.org/wiki/Special:Search" var="wikipediaUrl" encoding="UTF-8">
-            <sub:param name="search" value="\"${model.album}\""/>
+            <sub:param name="search" value="\"${fn:escapeXml(model.album)}\""/>
             <sub:param name="go" value="Go"/>
         </sub:url>
-        <sub:url value="allmusic.view" var="allmusicUrl">
-            <sub:param name="album" value="${model.album}"/>
+        <sub:url value="https://www.allmusic.com/search/albums/%22${fn:escapeXml(model.artist)}%22+%22${fn:escapeXml(model.album)}%22" var="allmusicUrl">
```

Comment:
This probably shouldn't be one big line with character encodings, it should use use a template substitution so it's understandable.